packages/core/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -62,19 +68,26 @@ export class DropdownMenu {
       ?.setAttribute("role", "menu");
     for (const ul of Array.from(this.host.getElementsByTagName("ul"))) {
       ul.setAttribute("aria-labelledby", this.button.id);
-      for (const li of Array.from(ul.getElementsByTagName("li"))) {
-        li.setAttribute("role", "menuitem");
-        if (li.classList.contains("dso-checked")) {
-          li.setAttribute("aria-selected", "true");
-        }
-      }
     }
 
     if (this.open) {
       this.openPopup();
     }
   }
 
+  componentWillRender() {
+    for (const li of Array.from(this.host.getElementsByTagName("li"))) {
+      for (const tab of tabbable(li)) {
+        tab.setAttribute("role", this.checkable ? "menuitemradio" : "menuitem");
```

Comment:
"menuitemradio" als het de linkvariant is, maar als het nu de buttonvariant is? Of weten we dat niet...

Reply:
Die keuze is (nog) niet te maken, want dat is niet afhankelijk van of je dat met een `a` of met een `button` doet. Het zal dan met een attribute op het component aangegeven moeten gaan worden. Vooralsnog houden we het dan maar op deze vorm.